Kickstarter-iOS/Views/Controllers/CommentRepliesViewController.swift

```diff
@@ -14,7 +14,7 @@ final class CommentRepliesViewController: UITableViewController {
   // MARK: Properties
 
   private let dataSource = CommentRepliesDataSource()
-  private let viewModel: CommentRepliesViewModelType = CommentRepliesViewModel()
+  internal let viewModel: CommentRepliesViewModelType = CommentRepliesViewModel()
```

Comment:
Made this internal (similar to the `CommentsViewController`) to enable testing the failed pagination in the view controller test file.